ISSUE_TEMPLATE.md

```diff
@@ -1,18 +1,20 @@
-For feature requests, please see: http://www.mattermost.org/feature-requests/.
-For troubleshooting questions, please post on the forums: http://forum.mattermost.org/.
+Per Mattermost guidelines, GitHub issues are for bug reports: http://www.mattermost.org/filing-issues/
 
-For issues with the product, please follow the steps at http://www.mattermost.org/filing-issues/ and post here in the following format:
+For troubleshooting see: http://forum.mattermost.org/.
+For feature proposals see: http://www.mattermost.org/feature-requests/
+
+If you've found a bug--obviously unintentional functionality--please report using the following format: 
 
 #### Summary
-Issue in one concise sentence
+Bug report in one concise sentence
 
 #### Steps to reproduce
 How can we reproduce the issue (what version are you using?)
 
 #### Expected behavior
 Describe your issue in detail
 
-#### Observed behavior
+#### Observed behavior (obviously unintentional) 
```

Comment:
@it33 If I was going to file a bug report and saw that it had to be "obviously unintentional functionality", I probably wouldn't file it: in case my issue isn't a bug report afterall, then I feel stupid because it "should have been obvious".

Also for a developer, an "obvious" bug might mean something different from a non-technical user.

Just my two cents. "Unintentional functionality" feels enough for me here.